src/Features/CSharp/Portable/InvertIf/CSharpInvertIfCodeRefactoringProvider.cs

```diff
@@ -113,6 +114,7 @@ protected override int GetJumpStatementRawKind(SyntaxNode node)
 
                 case SyntaxKind.LocalFunctionStatement:
                 case SyntaxKind.SetAccessorDeclaration:
+                case SyntaxKind.InitAccessorDeclaration:
                 case SyntaxKind.GetAccessorDeclaration:
                 case SyntaxKind.AddAccessorDeclaration:
                 case SyntaxKind.RemoveAccessorDeclaration:
```

Comment:
```cs
            switch (node)
            {
                case SwitchSectionSyntax:
                    return (int)SyntaxKind.BreakStatement;

                case LocalFunctionStatementSyntax:
                case AccessorDeclarationSyntax:
                case MemberDeclarationSyntax:
                case AnonymousFunctionExpressionSyntax:
                    return (int)SyntaxKind.ReturnStatement;

                case CommonForEachStatementSyntax:
                case DoStatementSyntax:
                case WhileStatementSyntax:
                case ForStatementSyntax:
                    return (int)SyntaxKind.ContinueStatement;
            }
```

Reply:
:memo: I just realized we have helpers for these. `IsContinuableConstruct` and `IsReturnableConstruct` could be used instead of both switches (excluding the switch-section case). Note the order is important since these may overlap. The current code is ok imo - up to the team if using the helper is preferable.